craco.config.js

```diff
@@ -50,6 +50,10 @@ module.exports = function (options) {
               urlPrefix: "~/static/js",
             })
           : null,
+        new webpack.ProvidePlugin({
```

Comment:
Can you check if we can use resolutions or possible peer deps instead oh having to expost React/ReactDOM?

Reply:
Managed to remove this :tada: